frontend/test/metabase-smoketest/user.cy.spec.js

```diff
@@ -220,15 +216,13 @@ describe("smoketest > new_user", () => {
     cy.findByText("Done").click();
     cy.findByText("Visualize").click();
 
-    cy.contains("ID");
+    cy.findByText("ID");
     cy.get(".Icon-table2");
-    cy.contains("Demo Column");
-    cy.contains("People").should("not.exist");
-
-    // =================
-    // should be able to use all notebook editor functions
-    // =================
+    cy.wait(1000).findByText("Demo Column");
+    cy.findByText("People").should("not.exist");
```

Comment:
Why are we testing that "People" doesn't exist?

Reply:
This should be checking "Products" doesn't exist (bc the table was just changed through the notebook editor). Fixing it now :)